examples/ethernet/ethernet/main/ethernet_main.c

```diff
@@ -164,8 +372,10 @@ void app_main()
     tcpip_adapter_init();
     esp_event_loop_init(NULL, NULL);
 
+#if (CONFIG_PHY_TLK110)
     eth_config_t config;
-    config.phy_addr = PHY31;
+//    config.phy_addr = PHY31;
+    config.phy_addr = PHY1; // tlk eval board
```

Comment:
The upstream repository uses PHY31 which would be specific to the original author and their hardware.  I changed it to the default for the evaluation boards for both the TLK110 and LAN8720.  I left the original commented out so the original author could easily change it.  Ideally, what I should do is place the PHY ID in the menuconfig file.